dobi.yaml

```diff
@@ -127,12 +127,21 @@ image=graph-descriptions-build:
   tags:
     - latest
 
+image=grapl-common-build:
+  image: grapl/grapl-common-python-build
+  context: src/python/grapl-common/
+  dockerfile: Dockerfile
+  target: grapl-common-python-build
+  tags:
```

Comment:
I believe we need a `depends` clause here e.g. ```
depends:
  - grapl-graph-descriptions-python-build``` because in the dockerfile you have ```COPY --from=grapl/grapl-graph-descriptions-python-build /home/grapl/venv venv```